File: src/SystemMessage/SystemMessage.java
package SystemMessage;

import Page.PageType;

public class SystemMessage {
	private FunctionType functionType;
	private PageType nextPageType;
	private Object input;
	
	
	public SystemMessage(FunctionType ft, PageType pt, Object i) {
		setFunctionType(ft);
		setNextPageType(pt);
		setInput(i);
	}


	public FunctionType getFunctionType() {
		return functionType;
	}
	
	public void setFunctionType(FunctionType functionType) {
		this.functionType = functionType;
	}

	public PageType getNextPageType() {
		return nextPageType;
	}

	public void setNextPageType(PageType nextPageType) {
		this.nextPageType = nextPageType;
	}
	
	public Object getInput() {
		return input;
	}

	public void setInput(Object input) {
		this.input = input;
	}


	
	
}

File: iter3/python_development/System/SystemMessage.py
from python_development.Page.PageType import PageType
from python_development.System.FunctionType import FunctionType

class SystemMessage:
    
    functionType = None
    nextPageType = None
    inputFromPAge = None

    def __init__(self, functionType, pageType, i):
        self.setFuncionType(functionType)
        self.setNextPageType(pageType)
        self.setInput(i)
        
    
    def getFunctionType(self):
        return self.functionType
    
    def setFuncionType(self, functionType):
        self.functionType = functionType

    def getNextPageType(self):
        return self.nextPageType
    
    def setNextPageType(self, nextPageType):
        self.nextPageType = nextPageType

    def getInput(self):
        return self.inputFromPAge
    
    def setInput(self, inputFromPAge):
        self.inputFromPAge = inputFromPAge